ambari-server/src/main/resources/common-services/ACCUMULO/1.6.1.2.2.0/metainfo.xml

```diff
@@ -195,8 +195,14 @@
       </commandScript>
 
       <requiredServices>
-        <service>ZOOKEEPER</service>
-        <service>HDFS</service>
+        <service>
+          <name>ZOOKEEPER</name>
+          <scope>INSTALL</scope>
+        </service>
+        <service>
+          <name>HDFS</name>
+          <scope>INSTALL</scope>
```

Comment:
Should we keep a type=LEGACY for old stack definitions?

Reply:
I don't think it's necessary, since for old stacks the LEGACY will be the same as the INSTALL type